utils/utils.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
AFAIK, if this file is renamed to `utils/index.js`, it could be used as `const utils = require(./utils)`

Reply:
Already pushed.